src/Fantomas/CodePrinter.fs

```diff
@@ -540,6 +547,11 @@ and genLetBinding astContext pref b =
             | _ ->
                 genPat astContext p
 
+        let isFunctionBinding =
+            match p with
```

Comment:
I'm also not sure if everything is cover with this pattern.
An active pattern seems to have a different [SynPat](https://fsharp.github.io/FSharp.Compiler.Service/reference/fsharp-compiler-syntaxtree-synpat.html).

F.ex. `let (|Add|_|) = Some bar`

Reply:
Destructured tuples is another case: `let (a,b) = foo`